addons/link_tracker/models/link_tracker.py

```diff
@@ -269,8 +270,8 @@ class LinkTrackerCode(models.Model):
 
     @api.model
     def _get_random_code_strings(self, n=1):
-        size = 3
         while True:
+            size = LINK_TRACKER_MIN_CODE_LENGTH
```

Comment:
Isn't it resetting the size to 3 every iterations, meaning the if below does not work anymore ?